### PR TITLE
feat: bandwidth stats

### LINKED ISSUE #2935: display waku upload/download bandwidth info in node tab
this task lays the groundwork for future work on the node tab & the community history serving section, first goal is to display the upload/download bandwidth stats

### PR DESCRIPTION
**Pull request by richard-ramos**
Fixes #2935

image

### CODE REVIEW

**Review by jrainville — APPROVED** [review]
Nice stuff

**Comment by iurimatias** [comment]
status-go pr https://github.com/status-im/status-go/pull/2286

File: ui/app/AppLayouts/Node/Rate.qml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                width: parent.width
+                readOnly: true
+                customHeight: 44
+                placeholderText: "0"
+                anchors.top: parent.top
+            }
+
+            StyledText {
+                color: Style.current.secondaryText
+                text: qsTrId("Kb/s")
```
Comment (pinebit) [inline ui/app/AppLayouts/Node/Rate.qml:41]: should not this be `qsTr()`?

File: ui/app/AppLayouts/Node/Rate.qml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
```
Comment (pinebit) [inline ui/app/AppLayouts/Node/Rate.qml:70]: should not this be qsTr()?

File: ui/app/AppLayouts/Node/Rate.qml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        text: qsTr("Bandwidth")
+        topPadding: Style.current.bigPadding
+        bottomPadding: Style.current.padding
+    }
+
+    Row {
+        width: parent.width
+        spacing: 10
+        StyledText {
+            text: "Upload"
```
Comment (pinebit) [inline ui/app/AppLayouts/Node/Rate.qml:21]: qsTr()

File: ui/app/AppLayouts/Node/Rate.qml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                text: qsTrId("Kb/s")
+                anchors.verticalCenter: parent.verticalCenter
+                anchors.right: uploadRate.right
+                anchors.rightMargin: Style.current.padding
+                font.pixelSize: 15
+            }
+        }
+
+        StyledText {
+            text: "Download"
```
Comment (pinebit) [inline ui/app/AppLayouts/Node/Rate.qml:50]: qsTr()

**Pull request merged**